client/dashboard/task-list/tasks/payments/stripe.js

```diff
@@ -20,24 +20,18 @@ import { WCS_NAMESPACE } from 'wc-api/constants';
 import withSelect from 'wc-api/with-select';
 import { PLUGINS_STORE_NAME } from '@woocommerce/data';
 
-/**
- * Internal dependencies
- */
-import { getCountryCode } from 'dashboard/utils';
-
 class Stripe extends Component {
 	constructor( props ) {
 		super( props );
 
 		this.state = {
-			autoConnectFailed: false,
+			oAuthConnectFailed: false,
 			connectURL: null,
 			errorTitle: null,
 			errorMessage: null,
```

Comment:
We can remove all references to `errorMessage` and `errorTitle` and its render now that autoconnect has been removed.

Reply:
Good catch! I've removed these along with the `renderErrorModal` function in https://github.com/woocommerce/woocommerce-admin/pull/4164/commits/94e6ce6eea0bf26144d61bb15161d07c5651e364